definitions/features/service.rb

```diff
@@ -44,8 +44,18 @@ def action_past_tense(action)
   end
 
   def filter_disabled_services!(action, service_list)
```

Comment:
Now that you no longer modify `service_list` in place, this no longer needs the `!`:
```suggestion
  def filter_disabled_services(action, service_list)
```